### ISSUE TITLE
Ability to run as a service

### ISSUE THREAD

**Issue by betullam** [ExLibrisGroup/alma-print-daemon#5]
**Our Goal:** We would like to run the Alma Print Daemon as a service on a server that manages our printers. It is a Windows Server 2016. We already tried installation and printing - that worked great.

**The Problem:** After logging off from the server the Alma Print Daemon is closed. Thus printing is not possible anymore. Due to our security policy every staff member that logs on to the server and does some work on it has to log off after he/she finished and therefore the Alma Print Daemon is closed also.

**Our Feature Request:** It would be great if the Alma Print Daemon could be run as a service that starts up when the Windows Server starts, without the need that a user logs in and starts the app manually. It would also be necessary that the app starts un-paused (see issue #1)

**Comment by fabian-combox** [comment]
What is the status on this? Have you already done anything?

**Comment by mgobat** [comment]
We are still investigating running the Alma Print Daemon as a Windows Service.

**Comment by betullam** [comment]
Any news on this?

**Comment by mgobat** [comment]
Coming very soon as a beta release on Windows.  It will require [nssm](https://nssm.cc/).

**Comment by betullam** [comment]
Great news, thank you!

**Comment by mgobat** [comment]
Please note that general release of 1.3 will come first.

**Comment by mgobat** [comment]
Alma Print Daemon 2.0-beta is available with this new feature.  

https://github.com/ExLibrisGroup/alma-print-daemon/releases/tag/v2.0.0-beta

**Comment by mgobat** [comment]
Has anyone been running the 2.0 beta as a service?  Any feedback?

**Comment by betullam** [comment]
@mgobat: Yes we are running it as a service and as far as I know it works without any bigger problems. But to be able to give you a correct answer I should ask the person in our IT department who installed it on our printer server. I will get back to you as soon as I have an answer.

**Comment by mgobat** [comment]
@betullam, thank you for your response. We look forward to any additional feedback from your IT department.

**Comment by betullam** [comment]
@mgobat The person from our IT department told me that it is running without problems as a Task, but that it fails when he tries to run it as a Service. I hope this helps!

**Comment by mgobat** [comment]
@betullam, could you please ask your IT person to send me some details on the task vs service experience? I'm at mark.gobat@exlibrisgroup.com.  Thank you.
